### PR TITLE
Add missing changelog items, fix UI glitches, supersede dependabot PRs

### LINKED ISSUE #4606: Markdown editor doesn't scroll well and hides the app bar (save and close buttons)
Steps to reproduce
1. open a file containing long text in the markdown editor
2.  scroll down using the scroll bar ***(doesn't show the end of the text)***
3. move to the end of the text using the cursor ***(hides app bar)***
4. move up to the start of the text ***(app bar is still not visible)***

Video Reference:

https://user-images.githubusercontent.com/52366632/104290603-a7d49100-54e2-11eb-83de-4a241e70931f.mp4

### LINKED ISSUE #6369: console log: Add Font Awesome attribution
## Expected outcome
The Web UI should output Font Awesome attribution in the console

## Background:
We should and must give proper attribution to the font awesome free icons
According to https://fontawesome.com/license/free we should include the attribution in the code (BOM follows in next sprints).
If you download `FA free web` (https://fontawesome.com/v5.15/how-to-use/on-the-web/setup/hosting-font-awesome-yourself) you'll find the `attribution.js` in the root of the zip file.

`attribution.js`

> ```
> console.log(`Font Awesome Free 5.15.4 by @fontawesome - https://fontawesome.com
> License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
> `) ```

## Not in this ticket

- SVGs should also contain attribution, will be done in ODS.
- Fullblown Bill of Materials will follow

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
PR to collect various things (missing changelog items, dropped commits from force-pushes, CSS one-line-fixes, translations improvements, ...) for the `v5.0.0` release

## Related Issue
- Works towards #6383
- Fixes #4606

### CODE REVIEW

File: packages/web-app-files/src/components/FilesList/ContextActions.vue [hunk 1]
```diff
@@ -178,6 +179,9 @@ export default {
   methods: {
     getSectionClasses(index) {
       const classes = []
+      if (this.menuSections.length === 1) {
```
Comment (kulmann) [inline packages/web-app-files/src/components/FilesList/ContextActions.vue:182]: By now I'd prefer a restructuring of all the different conditions into 0) early return, 1) handling of the paddings and 2) handling of the borders. If think what we want to have is:

## 0) early return
```js
if (!this.menuSections.length) {
  return classes
}
```

## 1) handling of paddings
```js
if (index < this.menuSections.length - 1) {
  classes.push('oc-pb-s')
}
if (index > 0) {
  classes.push('oc-pt-s')
}
```

Which means that all sections except the last get a bottom padding and all sections except the first get a top padding. For all sections in between the first and last sections this essentially boils down to `oc-py-s` but with less special case handling logic.

## 2) handling of borders
```js
if (index < this.menuSections.length - 1) {
  classes.push('oc-files-context-actions-border')
}
```

Which adds the bottom border for visual separation of sections to all sections except the last (and skips it entirely if you only have 1 section).

File: packages/web-runtime/themes/owncloud/theme.json [hunk 2]
```diff
@@ -118,7 +118,7 @@
         "swatch-warning-default": "",
         "swatch-warning-hover": "",
         "swatch-warning-muted": "",
-        "text-default": "#c2c2c2",
+        "text-default": "#DADCDF",
```
Comment (kulmann) [inline packages/web-runtime/themes/owncloud/theme.json:121]: we usually have lower case chars for color hex codes. of course not a technical issue at all, but not having it consistent makes me cringe a little bit 😁

**Review by kulmann — CHANGES_REQUESTED** [review]

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/22592/15/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptSharesToRoot-acceptShares_feature-L108.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/22602/41/1
The following scenarios passed on retry:
- webUISharingExternal/federationSharing.feature:387

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/22602/60/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:342

**Pull request merged**